common/src/main/java/com/wynntils/mc/mixin/CapeLayerMixin.java

```diff
@@ -40,4 +53,32 @@ private void render(
             ci.cancel();
         }
     }
+
+    @ModifyArg(
+            method = "render",
+            at =
+                    @At(
+                            value = "INVOKE",
+                            target =
+                                    "net/minecraft/client/renderer/MultiBufferSource.getBuffer(Lnet/minecraft/client/renderer/RenderType;)Lcom/mojang/blaze3d/vertex/VertexConsumer;"))
+    private RenderType setTranslucenceCapeRenderType(
+            RenderType original,
+            @Local(argsOnly = true) AbstractClientPlayer livingEntity,
+            @Local PlayerSkin playerSkin) {
+        Minecraft minecraft = Minecraft.getInstance();
+        boolean isBodyVisible = !livingEntity.isInvisible();
+        boolean translucent = !isBodyVisible && !livingEntity.isInvisibleTo(minecraft.player);
+
+        LivingEntityRenderTranslucentCheckEvent event =
```

Comment:
Does this mean this event is now called from 2 different sites? If so, is it called once for the entity itself, once for the cape layer? If so, it'd be wise to add a way for the event to specify which layer it's referring to, or alternatively, create a new, unique event.

Reply:
Done! I created a new event `PlayerFeatureRenderTranslucentCheckEvent` specific just for player. If somehow Wynncraft added an Elytra as a new armor or cosmetic, we can expand an additional event here.

Reply:
Also I reverted `RenderType` back to `entityTranslucent`, hope that should be enough for translucent cape.